tools/cli/adminCommands.go

```diff
@@ -186,28 +186,26 @@ func describeMutableState(c *cli.Context) *adminservice.DescribeMutableStateResp
 
 // AdminListNamespaces outputs a list of all namespaces
 func AdminListNamespaces(c *cli.Context) {
-	pFactory := CreatePersistenceFactory(c)
-	metadataManager, err := pFactory.NewMetadataManager()
-	if err != nil {
-		ErrorAndExit("Failed to initialize metadata manager", err)
-	}
+	adminClient := cFactory.AdminClient(c)
+	ctx, cancel := newContext(c)
+	defer cancel()
 
-	req := &persistence.ListNamespacesRequest{
-		PageSize: c.Int(FlagPageSize),
+	req := &adminservice.GetNamespacesRequest{
+		PageSize: int32(c.Int(FlagPageSize)),
 	}
 	paginationFunc := func(paginationToken []byte) ([]interface{}, []byte, error) {
 		req.NextPageToken = paginationToken
-		response, err := metadataManager.ListNamespaces(req)
+
+		resp, err := adminClient.GetNamespaces(ctx, req)
```

Comment:
the API is a list API right? so `GetNamespaces` -> `ListNamespaces` ?

Reply:
I think it is not worth to make this breaking change.

Reply:
done